packages/core/src/dynamoDb/vue/tableView.ts

```diff
@@ -0,0 +1,79 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as nls from 'vscode-nls'
+import { ExtContext } from '../../shared'
+import { VueWebview } from '../../webviews/main'
+import { getLogger, Logger } from '../../shared/logger'
+import { telemetry } from '../../shared/telemetry/telemetry'
+import { DynamoDbTableNode } from '../explorer/dynamoDbTableNode'
+import { getTableContent, RowData, TableData } from '../utils/dynamodbUtils'
+import { Key, ScanInput } from 'aws-sdk/clients/dynamodb'
+
+const localize = nls.loadMessageBundle()
+
+interface DynamoDbTableData {
+    TableName: string
+    Region: string
+    currentPage: number
+    tableContent: RowData[]
+    tableHeader: RowData[]
+    lastEvaluatedKey?: Key
+}
+
+export class DynamoDbTableWebview extends VueWebview {
+    public static readonly sourcePath: string = 'src/dynamoDb/vue/index.js'
+    public readonly id = 'dynamoDbTableView'
+
+    public constructor(private readonly data: DynamoDbTableData) {
+        super(DynamoDbTableWebview.sourcePath)
+    }
+
+    public init() {
+        telemetry.schemas_view.emit({ result: 'Succeeded' })
```

Comment:
is this telemetry relevant for this? I thought schemas_view would be for something seperate

Reply:
Updated